programs/pkey/gen_key.c

```diff
@@ -248,7 +248,7 @@ int main(int argc, char *argv[])
     mbedtls_mpi_init(&D); mbedtls_mpi_init(&E); mbedtls_mpi_init(&DP);
     mbedtls_mpi_init(&DQ); mbedtls_mpi_init(&QP);
 #endif /* MBEDTLS_RSA_C */
-
+	mbedtls_entropy_init(&entropy);
```

Comment:
I think it's best to keep the blank line here, and also the indentation should match the surrounding lines - we use four spaces, rather than a tab character